packages/performance/docs/events.md

```diff
@@ -0,0 +1,126 @@
+# Events
+
+## Lifecycle Events
+
+### Time to First Byte
+
+Use enum `EventType.TimeToFirstByte`
```

Comment:
> Use enum `EventType.TimeToFirstByte`

I find these statements confusing.  I'd prefer if we started this entire section of the docs explaining that there is an enum called `EventType` that all events have a status from, and then title the individual sections like:

```
### EventType.TimeToFirstBye

The time... (rest of the explanation)
```

Reply:
Yeah, it's good to say that

> The `type` field on the event will be `EventType.TimeToFirstByte`

But the more important bit is a brief explanation and the link to learn about the event (also, weird that the link is just `Learn more`, no sentence, at the end of each section)

Reply:
I would like the title to be a part of explanation rather just the type so the doc can still read like an article.
However, I still want to keep the type for the ease of searching later.

What if I do this structure? 

```
##### Time to First Byte (`EventType.TimeToFirstByte`)
The time until the server sends the first part of the response.
Learn more about this metric [here](https://developers.google.com/web/tools/chrome-devtools/network-performance/understanding-resource-timing).
```

Reply:
In general, avoid using `here` as the linked word (bad for screen reader users). Otherwise, structure LGTM 👍 (of note, it might cause the anchor links for those titles to be kind of janky, but oh well)